Recurly/BaseClient.cs

```diff
@@ -83,6 +88,7 @@ public int GetResourceCount(string url, Dictionary<string, object> queryParams)
             return int.Parse(recordCount);
         }
 
+        [ExcludeFromCodeCoverage]
```

Comment:
Why are we excluding from Code Coverage?

Reply:
This `_` in the method name denotes it's not part of the public API. I could go either way on this, but it's technically not something that we need coverage for.

Reply:
`[SECURITY WARNING] _SetApiUrl is for testing only and not supported in production.`

Primarily because this method is 100% for internal smoke testing purposes.

Reply:
sounds good.